layouts/shortcodes/cve-feed.html

```diff
@@ -0,0 +1,19 @@
+<table class="security-cves">
+  <caption>{{ T "cve_table" }}</caption>
+  <thead>
+  <tr>
+    <th>{{ T "cve_id" }}</th>
+    <th>{{ T "cve_summary"}}</th>
+    <th>{{ T "cve_issue_url" }}</th>
+  </tr>
+  </thead>
+  <tbody>
+  {{ range $issues := getJSON .Site.Params.cveFeedBucket }}
+  <tr>
+    <td><a href="{{ .cve_url }}">{{ .cve_id | htmlEscape }}</a></td>
+    <td>{{ .summary | htmlEscape }}</td>
```

Comment:
These might need to be:
```suggestion
   <td><a href="{{ .cve_url }}">{{ .cve_id | htmlEscape | safeHTML }}</a></td>
   <td>{{ .summary | htmlEscape | safeHTML }}</td>
```

Look for `&gt;` and `&lt;` in the preview. Marking these escaped strings as safe should fix that (untested).